README.md

```diff
@@ -101,8 +101,8 @@ module.exports = {
 		API_PATH: 'api',
 		SERVICES: {
 			openidconnect: 'openidconnect',
-			asab_config: 'asab_config',
-			seacat_auth_webui: 'http://localhost:3000/auth'
+			asab-config: 'asab-config',
+			seacat-auth-webui: 'http://localhost:3000/auth'
```

Comment:
@queuing4oranges Since you've changed docs and readme, you should also change it here:

https://github.com/TeskaLabs/asab-webui/blob/refactor/update-service-names/src/modules/auth/header.js#L40-L41

:)